Added an example for Targeting block (#403)
ezsystems/user-documentation: docs/content_management/block_reference.md

- **Name** – Enter a name for the page block.
- **Product list** — Create a list of products. Either enter a comma-separated list of product codes and click **Add**, or click **Select products**, select products, and click **Confirm**.

> ### Product embed

Author: Done on purpose, there's a small bug here:
https://doc.ibexa.co/projects/userguide/en/latest/content_management/block_reference/

The rules are checked in order, so when a user belongs to more than one segment, the first rule applies.

For example, consider the following configuration:

| Segment   | Content displayed |
|-----------|-------------------|
| Segment A | Content 1         |
| Segment B | Content 2         |
| Segment C | Content 3         |

> To a user who belongs to **Segment A**, **Segment B**, and **Segment C**, is displayed **Content 1**, because the block evaluates rules from top to bottom and stops at the first match.

Author: ```suggestion
To a user who belongs to **Segment A**, **Segment B**, and **Segment C**, **Content 1** is displayed, because the block evaluates rules from top to bottom and stops at the first match.
```

> results in displaying **Content 1**, because the block evaluates rules from top to bottom and stops at the first match.

Reviewer: The table cutting the sentence results in a weird reading.

```suggestion
For example, consider the following configuration:

| Segment   | Content displayed |
|-----------|-------------------|
| Segment A | Content 1         |
| Segment B | Content 2         |
| Segment C | Content 3         |

To a user who belongs to **Segment A**, **Segment B**, and **Segment C**, is displayed **Content 1**, because the block evaluates rules from top to bottom and stops at the first match.
```

Author: Thank you! 🙇

> Even though the user also belongs to **Segment B** and **Segment C**, those rules are never evaluated.

Reviewer: Let's be clear about they'ren't evaluated for this example user
```suggestion
Even though the user also belongs to **Segment B** and **Segment C**, those rules are never evaluated in this case.
```
